src/helm/benchmark/presentation/summarize.py

```diff
@@ -251,7 +251,43 @@ def compute_aggregate_row_win_rates(table: Table, aggregation: str = "mean") ->
     return aggregate_win_rates
 
 
+def compute_aggregate_row_means(table: Table) -> List[Optional[float]]:
+    """
+    Computes the aggregate mean of each row across columns.
+    Returns a list of means, one per row, with None if a row was never meaningfully comparable (i.e., all
+    non-null values of the row are in columns we skip).
+    """
+
+    means_per_row: List[Optional[float]] = []
+
+    # check for all header cells where specified, that lower_is_better is consistent
+    orderings = []
+    for elem in table.header:
+        if elem.lower_is_better is not None:
+            orderings.append(elem.lower_is_better)
```

Comment:
I think you should actually include these, because `None` means that there is no directionality for the metric (e.g. number of prompts).